packages/ui/src/validators/components/ValidatorsList.tsx

```diff
@@ -65,7 +65,7 @@ export const ValidatorsList = ({ validators, eraIndex, order, pagination }: Vali
                     The APR is subject to the amount staked and have a diminishing return for higher token amounts. This
                     is calculated as follow:
                     <br />
-                    <code>Yearly Reward * Commission / Stake</code>
+                    <code>Yearly Reward * (1 - Commission) / Stake</code>
```

Comment:
Are you sure about that ? It should show the nominator reward not the validator one. I might be wrong though based on @ivanturlakov's comments but could you detail why it should be `Reward * (1 - Commission) / Stake` please ?

Reply:
I have checked, please check here.
https://handbook.joystream.org/system/validation

Reply:
https://handbook.joystream.org/system/validation#calculating-rewards